rustfmt.toml

```diff
@@ -1,5 +1,4 @@
 unstable_features = true
 max_width = 79
 reorder_imports = true
-fn_args_layout = "Tall"
```

Comment:
this appears to be a no-op change

Reply:
Mmm, will give it a check.

Reply:
The difference is the missing newline in the new file, let's just revert this.